### DISCUSSION TITLE
Achievments

### DISCUSSION THREAD

**Discussion by vverenich** [Schweinepriester/github-profile-achievements#457] [Q&A] [answered]
Can I get the Galaxy Brain achievement?

**Comment by Foltrex** [comment, marked as answer]
Yes, you can

**Comment by d4tdev** [comment]
You can take more Galaxy Brain by answer question here
https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions?discussions_q=is%3Aunanswered

He will mark your answer if this answer is true

**Reply by muhammadnurulahsan** [reply]
How do I get the Galaxy Brain badge?

**Comment by haithamaouati** [comment]
If my answer helped you, you could mark it as answered. Please.

**Reply by muhammadnurulahsan** [reply]
How do I get the Galaxy Brain badge?

**Comment by muhammadnurulahsan** [comment]
How do I get the Galaxy Brain badge?
